docs/applications/harness-application.md

```diff
@@ -17,14 +17,16 @@ positional arguments:
 optional arguments:
   -h, --help            show this help message and exit
   -t TEMPLATES, --template TEMPLATES
-                        Add a template name. Available templates: - base (backend flask app based on openapi, always included) - webapp (webapp including backend and frontend) - db-postgres - db-neo4j - db-mongo
+                        Add a template name. Available templates: - base (always included) - server (backend flask app based on openapi) - webapp (webapp including backend and frontend) - db-postgres - db-neo4j - db-mongo - django-app (fastapi django backend based on openapi)
```

Comment:
does the django app include also a frontend webapp? I'm also for renaming the server name to be flask something to be clear, no need to keep the old name

Reply:
yes the django app add the webapp automatically

Reply:
I renamed server to flask-server